examples/with_winit/src/lib.rs

```diff
@@ -136,397 +136,412 @@ fn run(
+                                Key::Named(NamedKey::ArrowLeft) => {
+                                    scene_ix = scene_ix.saturating_sub(1)
                                 }
-                            }
-                            Some(VirtualKeyCode::Space) => {
-                                transform = Affine::IDENTITY;
-                            }
-                            Some(VirtualKeyCode::S) => {
-                                stats_shown = !stats_shown;
-                            }
-                            Some(VirtualKeyCode::D) => {
-                                complexity_shown = !complexity_shown;
-                            }
-                            Some(VirtualKeyCode::C) => {
-                                stats.clear_min_and_max();
-                            }
-                            Some(VirtualKeyCode::M) => {
-                                aa_config_ix = if modifiers.shift() {
-                                    aa_config_ix.saturating_sub(1)
-                                } else {
-                                    aa_config_ix.saturating_add(1)
-                                };
-                            }
-                            Some(VirtualKeyCode::P) => {
-                                if let Some(renderer) = &renderers[render_state.surface.dev_id] {
-                                    if let Some(profile_result) = &renderer
-                                        .profile_result
-                                        .as_ref()
-                                        .or(profile_stored.as_ref())
+                                Key::Named(NamedKey::ArrowRight) => {
+                                    scene_ix = scene_ix.saturating_add(1)
+                                }
+                                Key::Named(NamedKey::ArrowUp) => complexity += 1,
+                                Key::Named(NamedKey::ArrowDown) => {
+                                    complexity = complexity.saturating_sub(1)
+                                }
+                                Key::Character(key @ "a") | Key::Character(key @ "e") => {
+                                    if let Some(prior_position) = prior_position {
+                                        let is_clockwise = key == "e";
+                                        let angle = if is_clockwise { -0.05 } else { 0.05 };
+                                        transform = Affine::translate(prior_position)
+                                            * Affine::rotate(angle)
+                                            * Affine::translate(-prior_position)
+                                            * transform;
+                                    }
+                                }
+                                Key::Named(NamedKey::Space) => {
+                                    transform = Affine::IDENTITY;
+                                }
+                                Key::Character("s") => {
```

Comment:
Same concern about caps lock for all of these

Reply:
So regarding caps lock: I didn't consider it even, thinking winit would ignore modifiers, and act like it used to before.
I reworked the code: Winit does expose a way to ignore modifiers on key via a `KeyEventExtModifierSupplement` trait, however oddly enough its not exposed for a web target, so we cannot use it here, so now will just use `to_lowercase`.